plugins/modules/ntnx_karbon_clusters_info.py

```diff
@@ -0,0 +1,224 @@
+      nutanix_username: "{{ username }}"
+      nutanix_password: "{{ password }}"
+      validate_certs: False
+      cluster_name: "cluster-name"
+      fetch_kubeconfig: true
+    register: result
+"""
+RETURN = r"""
+cni_config:
+  description: Container networking interface (CNI) information.
+  returned: always
+  type: dict
+  sample:
+    {
+                "flannel_config": null,
+                "node_cidr_mask_size": 24,
+                "pod_ipv4_cidr": "172.20.0.0/16",
+                "service_ipv4_cidr": "172.19.0.0/16"
+            }
+etcd_config:
+  description: Etcd configuration information.
+  returned: always
+  type: dict
+  sample: {
+                "node_pools": [
+                    "test-module21_etcd_pool"
+                ]
+            }
+kubeapi_server_ipv4_address:
+    description: IPV4 address of the API server.
+    returned: always
+    type: str
+    sample: "10.44.78.171"
+master_config:
+    description: Configuration of master nodes.
+    returned: always
+    type: dict
+    sample: {
+                "deployment_type": "single-master",
+                "node_pools": [
+                    "test-module21_master_pool"
+                ]
+            }
+name:
+    description: K8s cluster name.
+    returned: always
+    type: str
+    sample: "test-module21"
+status:
+    description: K8s cluster status.
+    returned: always
+    type: str
+    sample: "kActive"
+uuid:
+    description: The universally unique identifier (UUID) of the k8s cluster.
+    returned: always
+    type: str
+    sample: "00000000-0000-0000-0000-000000000000"
+version:
+    description: K8s version of the cluster.
+    returned: always
+    type: str
+    sample: "1.19.8-0"
+worker_config:
+    description: Worker configuration information.
+    returned: always
+    type: dict
+    sample: {
+                "node_pools": [
+                    "test-module21_worker_pool"
+                ]
+            }
+kube_config:
+    description: write
+    returned: if fetch_kubeconfig is true
+    type: str
+certificate:
+    description: ssh certifcate
+    returned: if fetch_ssh_credentials is true
+    type: str
+expiry_time:
+    description: expire time for certificate
+    returned: if fetch_ssh_credentials is true
+    type: str
+    sample: "2022-08-16T06:33:18.000Z"
+private_key:
+    description: user private key
+    returned: if fetch_ssh_credentials is true
+    type: str
+username:
+    description: name
+    returned: if fetch_ssh_credentials is true
+    type: str
+    sample: admin
+"""
+
+from ..module_utils.base_info_module import BaseInfoModule  # noqa: E402
+from ..module_utils.karbon.clusters import Cluster  # noqa: E402
+
+
+def get_module_spec():
+
+    module_args = dict(
+        cluster_name=dict(type="str"),
+        fetch_ssh_credentials=dict(type="bool"),
+        fetch_kubeconfig=dict(type="bool"),
+    )
+
+    return module_args
+
+
+def get_cluster(module, result):
+    cluster = Cluster(module)
```

Comment:
Can we add a process where they can get both ssh credentials and kubeconfig together ?